src/Core/utils/label.js

```diff
@@ -1,3 +1,109 @@
+export function returnWrapNonLabelsSugarFunction({
```

Comment:
We can think about what we might want to unit test for a function like this as we merge in the jest runner. Would this be too small?

Reply:
Good question.  Seems small.